README.md

```diff
@@ -8,7 +8,7 @@ This project is not affiliated, associated, authorized, endorsed by, or in any w
 
 ## Features
 
-- Support for Push and Delivery to multiple Event Grid topics by sending EventGridEvents (Push) on Custom topics to `http://127.0.0.1:6500/<topic-name>/api/events` and CloudEvents (Pull) on Namespace topics to `http://127.0.0.1:6500/topics/<topic-name>:publish`.
+- Support for Push and Delivery to multiple Event Grid topics by sending EventGridEvents/CloudEvents (Push) on Custom topics to `http://127.0.0.1:6500/<topic-name>/api/events` and CloudEvents (Pull) on Namespace topics to `http://127.0.0.1:6500/topics/<topic-name>:publish`.
```

Comment:
- Support publishing events to Custom Topics (EventGridEvents/CloudEvents) and Namespace Topics (only CloudEvents)
- Support Push & Pull Delivery Model

Pas besoins de emttre les liens ici si on les mentionne plus bas